docs/openstack/edpm_adoption.md

```diff
@@ -160,6 +162,9 @@ done
       - install-os
       - configure-os
       - run-os
+      - libvirt
+      - nova
+      - nova-compute-extraconfig
```

Comment:
The `nova-compute-extraconfig` is enough we don't need two nova ansible runs in the same deployment. The run of the second overwrites  anything that was done by the first.

Reply:
done